packages/ado-extension/src/progress-reporter/pull-request/ado-pull-request-comment-creator.ts

```diff
@@ -177,6 +181,12 @@ export class AdoPullRequestCommentCreator extends ProgressReporter {
         }
     }
 
+    private failOnBaselineNotUpdated(suggestedBaselineUpdate: null | BaselineFileContent): void {
+        if (this.adoTaskConfig.getBaselineFile() !== undefined && suggestedBaselineUpdate !== null) {
+            throw 'Failed: New accessibility errors found, not in the baseline. See PR comment for more info.';
```

Comment:
2 thoughts:
1. This error seems oddly stilted and may need some wordsmithing
2. Is this the error message that also gets displayed when errors in the baseline go away (i.e., are fixed) ? If so, maybe a more generic error would make sense, such as 
```suggestion
            throw 'Failed: The baseline file needs to be updated. See the PR comments for more details.';
```